project/plugins.sbt

```diff
@@ -1,3 +1,5 @@
 addSbtPlugin("com.47deg"  % "sbt-microsites" % "0.7.4")
 
 addSbtPlugin("com.scalapenos" % "sbt-prompt" % "1.0.2")
+
+addSbtPlugin("org.ensime" % "sbt-ensime" % "2.0.1")
```

Comment:
It was actually a mistake for me to commit `ensime.sbt`, it should be removed instead of adding the plugin.

Reply:
Fixed.